docs/topics/api/overview.rst

```diff
@@ -261,6 +259,17 @@ The downside of using the `v3` API is, of course, no new cool features!
 The documentation for `v3` can be accessed at: :ref:`v3-api-index`
 
 
+.. _api-experimental-v5:
+
+----------------------------------------
+Extra-Experimental/Future-default v5 API
```

Comment:
That title is… interesting, I can't really decipher what this means to me tbh. We only have one "experimental" API now, which is V5 after merging this right?

Reply:
We call the `v4` experimental all over the place and I wasn't sure how we wanted to brand the `v4` API at this point.

Reply:
V5 -> Experimental
V4 -> Default, Stable
V3 -> Deprecated

Maybe?

Reply:
That's ideally where we'd be, yeah, but whether v4 is completely stable when we're changing things to support addons-frontend changes, and whether v3 can be deprecated while release Firefox is on it are open questions.  https://github.com/mozilla/addons-server/issues/11273 has all steps.  
Moving discussion of changes to https://github.com/mozilla/addons-server/issues/11361

Reply:
addressed in https://github.com/mozilla/addons-server/pull/11369